traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -31,22 +34,39 @@ open class AvailabilityViewController: UIViewController {
     }
     var selectedOption: BookingOption?
     var optionsViewController: BookingOptionsViewController?
+    var numberOfMonths = 12
 
     private var passes: [Pass]?
-
-    private var datePickerCellVisible = false
-    private var dateCellIndexPath: IndexPath {
-        return IndexPath(row: 0, section: 0)
-    }
-
-    private var datePickerCellIndexPath: IndexPath {
-        return IndexPath(row: 1, section: 0)
+    private var pageController: UIPageViewController?
+    private var calendarViewControllers: [Date: AvailabilityCalendarViewController] = [:]
```

Comment:
Poor naming convention! it doesnt make sense to have a dictionary called "calendarViewControllers" where its items have "availabilityCalendarViewController"  Availability != (just) calendar. it should be "calendarViewControllers": [Data: CalendarViewController]. When your naming convention stops making sense, and your forced to mix stuff like this, it is normally an indication of two things 1) You're not naming things correctly 2) You've implemented a bad architecture and your controllers are doing too much. In this case it is the second.